src/parser/xasl_generation.c

```diff
@@ -27546,6 +27644,28 @@ pt_make_sq_cache_key_struct (QPROC_DB_VALUE_LIST key_struct, void *p, int type)
 	      cnt += ret;
 	    }
 	}
+      if (xasl_src->outptr_list)
+	{
+	  regu_var_list_p = xasl_src->outptr_list->valptrp;
+	  for (i = 0; i < xasl_src->outptr_list->valptr_cnt; i++)
+	    {
+	      if (!regu_var_list_p)
+		{
+		  return ER_FAILED;
```

Comment:
```suggestion
		  assert (false);
		  return ER_FAILED;
```
`valptr_cnt`와 실제 개수가 일치하지 않는 것은 불가능하다고 생각합니다.